docs/content.zh/docs/ops/metrics.md

```diff
@@ -840,7 +840,7 @@ Deprecated: use [Default shuffle service metrics](#default-shuffle-service)
     </tr>
     <tr>
       <td>outPoolUsage</td>
-      <td>An estimate of the output buffers usage.</td>
+      <td>An estimate of the output buffers usage, the pool usage was > 100% if overdraft buffers are being used.</td>
       <td>Gauge</td>
     </tr>
     <tr>
```

Comment:
I think in both English and Chinese versions there is also a second occurrence of the same metric for the `Default shuffle service metrics` that you also should modify. Here you have only modified the docs for the deprecated component that is preserved for the backward compatibility reasons.

Reply:
Thanks for your reminder, updated.